test/topology/util.py

```diff
@@ -477,7 +477,9 @@ async def new_test_keyspace(manager: ManagerClient, opts, host=None):
         async with new_test_keyspace(cql, '...') as keyspace:
     """
     keyspace = unique_name()
-    await manager.get_cql().run_async("CREATE KEYSPACE " + keyspace + " " + opts, host=host)
+    # Use CREATE KEYSPACE IF NOT EXISTS as a workaround for
+    # https://github.com/scylladb/python-driver/issues/317
```

Comment:
Should this be repeated at the other places above?
BTW why not convert those to use `new_test_keyspace()`?

Reply:
What other places exactly?
For creating tables / materialized view?
I'm not sure if we've seen the same issue not for keyspaces

Reply:
There are two more changes in this same file (in the same commit). Those are the ones I refer to.